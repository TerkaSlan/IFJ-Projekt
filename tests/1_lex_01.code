/*This test should throw error because in name of class 1SomeClass starts with number */

class Main
{
	static void run()
	{
		1Someclass.someFunction(10,15);
	} // end of static void run()
	
} // end of class Main


class 1Someclass
{
	/*Adds two numbers*/
	static int someFunction(int x, int y)
	{
		return (x+y);
	}
}