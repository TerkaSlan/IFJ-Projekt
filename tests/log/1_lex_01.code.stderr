ERROR: Syntactic - Line: 7 - Unexpected token.
